### PR TITLE
Add receive

### PR DESCRIPTION
**Pull request by sarahsturgeon**

### CODE REVIEW

File: moon/cfc_chat_transit/client/receive_remote_message.moon [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import Start, Receive, ReadBool, ReadString, WriteBool, Send from net
+import AddToolCategory, AddToolMenuOption from spawnmenu
+
+shouldReceiveRemoteMessages = CreateConVar "cfc_chat_transit_remote_messages", true, FCVAR_ARCHIVE, "Should receive remote messges in chat", 0, 1
+
+colors =
+    white: Color 255, 255, 255
+    blurple: Color 114, 137, 218
```
Comment (samuelWilliams99) [inline moon/cfc_chat_transit/client/receive_remote_message.moon]: nice

File: moon/cfc_chat_transit/server/remote_messages.moon [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import AddNetworkString from util
+import Start, Receive, ReadBool, WriteString, Send from net
+
+AddNetworkString "CFC_ChatTransit_RemoteMessagePreference"
+AddNetworkString "CFC_ChatTransit_RemoteMessageReceive"
+
+IsValid = IsValid
+recipients = RecipientFilter!
```
Comment (samuelWilliams99) [inline moon/cfc_chat_transit/server/remote_messages.moon:10]: woah thats neat, didnt know about that

**Review by samuelWilliams99 — APPROVED** [review]
Looks neat! Very cool project

File: moon/cfc_chat_transit/server/remote_messages.moon (hunk 2 above)
Comment (sarahsturgeon) [reply moon/cfc_chat_transit/server/remote_messages.moon:10]: Me neither!

**Pull request merged**